docs/src/apps/breaking-changes.md

```diff
@@ -0,0 +1,91 @@
+---
+title: API Breaking Changes
+---
+
+As the Solana developer ecosystem grows, so does the need for clear expectations around
+breaking API and behavior changes affecting applications and tooling built for Solana.
+In a perfect world, Solana development could continue at a very fast pace without ever
+causing issues for existing developers. However, some compromises will need to be made
+and so this document attempts to clarify and codify the process for new releases.
+
+Solana API's, SDK's, and CLI tooling are typically tied to validator software releases.
+Validator software releases DO NOT strictly follow semantic versioning.
+
+### Release Cadence
+
+The Solana RPC API, Rust SDK, CLI tooling, and BPF Program SDK are all updated and shipped
+along with each Solana validator release and should always be compatible between patch
+updates of a particular minor version release.
+
+#### Minor Releases (1.x.0)
+
+Experimental changes and new proposal implementations are added to new MINOR version
+releases (ex. 1.4.0) and are first run on Solana's Tour de SOL testnet cluster. After
+those changes have proven to be stable, the Mainnet Beta cluster will be updated to the
+new version.
+
+#### Patch Releases (1.0.x)
+
+Low risk features, non-breaking changes, and security and bug fixes are shipped as part
+of PATCH version releases (ex. 1.0.11).
+
+### RPC API
```

Comment:
How about a section for the SDK and program interfaces:

### Program SDK

Patch releases:
- New APIs
- Bug fixes
- Performance improvements
- Security fixes

Minor releases:
- Removal of deprecated APIs
- Backward compatibility breaking changes